db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_09_30_053037) do
+ActiveRecord::Schema[7.1].define(version: 2023_09_30_053037) do
```

Comment:
interesting change - not related to the goal of the PR I think. Possibly a good idea to merge in general tho, it may have been left out of an earlier unrelated PR?